<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtncuentas</class>
 <widget class="QMainWindow" name="vtncuentas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>511</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>144</green>
        <blue>134</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>144</green>
        <blue>134</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>144</green>
        <blue>134</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>144</green>
        <blue>134</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Cuentas</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logobanco.png</normaloff>logobanco.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblTituloU">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>BANCA VIRTUAL</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSubTitulo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CUENTAS</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLogoU">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logobanco.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="btnModificar">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>460</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>MODIFICAR</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableCuentas">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>991</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>46</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>140</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>CBU</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ALIAS</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>N° CLIENTE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TIPO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>FECHA CREACION</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>BALANCE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CLIENTE</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnCrear">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>460</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string>AÑADIR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leBuscar">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>130</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEliminar">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>460</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>ELIMINAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBuscar">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>140</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Buscar:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnActualizar">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Actualizar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>actualizar.png</normaloff>actualizar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>38</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEstadisticas">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 0, 255);</string>
    </property>
    <property name="text">
     <string>ESTADISTICAS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBuscar">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>130</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>buscar.jpg</normaloff>buscar.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>38</width>
      <height>38</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
